### DISCUSSION TITLE
Explorer Taskbar Fix

### DISCUSSION THREAD

**Discussion by WashierPlayz** [vrcx-team/VRCX#911] [Ideas]
I noticed that sometimes, VRChat freezes the explorer bar and can't select certain things until PC restart, but hitting Ctrl+Alt+Del and then canceling fixes it, maybe implementing something similar to fix could be nice?

**Comment by loukylor** [comment]
VRChat? Do you mean VRCX? Also what do you mean freezes the explorer bar? Like the search bar on the bottom left of windows? Or the path bar in file explorer?

**Reply by WashierPlayz** [reply]
I mean the entire Windows Taskbar, and some Applications I can't interact with when it does freeze

**Reply by Myrkie** [reply]
> I mean the entire Windows Taskbar, and some Applications I can't interact with when it does freeze

that's more of an issue with base windows, brand new install of windows will do this, best you can do is restart the taskbar process "explorer.exe" this is not a VRCX related issue, nor something we can easily fix ourselves.
